packages/platform-browser/src/hydration.ts

```diff
@@ -127,6 +127,17 @@ function provideZoneJsCompatibilityDetector(): Provider[] {
  * passing special functions (from the `HydrationFeatures` set) as arguments to the
  * `provideClientHydration` function.
  *
+ * By default, the [`provideClientHydration()`](api/platform-browser/provideClientHydration) function enables the recommended set of features for the optimal performance for most of the applications. It includes the following features:
```

Comment:
The "by default" part is already included into a paragraph above, so we can update this as:
```suggestion
 * passing special functions (from the `HydrationFeatures` set) as arguments to the
 * `provideClientHydration` function. It includes the following features:
```